app/models/profile.rb

```diff
@@ -0,0 +1,14 @@
+class Profile < ApplicationRecord
+  belongs_to :user
+
+  # This method generates typed accessors for all active profile fields
+  def self.define_store_accessors!
```

Comment:
Originally I executed the body directly, but making it a method and calling it right away makes this testable and is also useful when trying things out in a `rails console` session etc.